packages/pds/src/db/tables/label.ts

```diff
@@ -0,0 +1,12 @@
+export const tableName = 'label'
+
+export interface Label {
+  sourceDid: string
+  subjectUri: string
+  subjectCid: string
+  value: string
+  negated: 0 | 1 // @TODO convert to boolean in app-view
```

Comment:
Is this todo still outstanding?

Reply:
it's for when we put this on the AppView & we're using postgres which allows for booleans. using int2 here cuz sqlite